__tests__/components/editor/property/LiterTypeLabel.test.js

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 Stanford University see LICENSE for license
+
+import React from "react"
+import { render } from "@testing-library/react"
+import { createState } from "stateUtils"
+import configureMockStore from "redux-mock-store"
+import LiteralTypeLabel from "components/editor/property/LiteralTypeLabel"
+import { Provider } from "react-redux"
+
+const mockStore = configureMockStore()
+
+describe("<LiteralTypeLabel />", () => {
+  it("displays the correct text for unvalidated literal", () => {
+    const state = createState({ hasTwoLiteralResources: true })
+    const store = mockStore(state)
+    const propertyTemplate =
+      state.entities.propertyTemplates[
+        "ld4p:RT:bf2:Title:AbbrTitle > http://id.loc.gov/ontologies/bibframe/mainTitle"
+      ]
+    const label = render(
```

Comment:
Can you use `renderComponent()` from `testUtils`? It will take care of the provider for you.

Reply:
ok, found this pattern in a few other tests, will do a separate PR for those